definition.yml

```diff
@@ -510,3 +510,15 @@ repos:
       - config-store
     environment:
       NATS_URI: 'nats://nats:4222'
+
+  - name: logger
+    path: git@github.com:ernestio/logger.git
+    branch: master
+    links:
+      - nats
+    environment:
+      NATS_URI: 'nats://nats:4222'
+      ERNEST_LOG_FILE: '/tmp/ernest.log'
+    volumes:
+      - /tmp/:/tmp/
```

Comment:
Can you set this to be something inside the cloned ernest directory please? something like ./logs/:/tmp/.
We don't want to expose /tmp/ to the container if we can help it.